policy-controller/k8s/status-controller/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+use linkerd_policy_controller_k8s_api as k8s;
+use linkerd_policy_controller_k8s_index as index;
+use tokio::sync::mpsc;
+
+pub async fn process_patches(
+    client: k8s::Client,
+    mut patches_rx: mpsc::UnboundedReceiver<index::Patch>,
+) {
+    let patch_params = k8s::PatchParams::apply("policy.linkerd.io");
+    while let Some(patch) = patches_rx.recv().await {
+        let index::Patch {
+            name,
+            namespace,
+            value,
+        } = patch;
+        tracing::info!(%value, "Patching HTTPRoute");
```

Comment:
nit: is it worth adding 
```suggestion
        tracing::info!(%name, %namespace, %value, "Patching HTTPRoute");
```
here? or are they already incldued in the JSON?